src/BackpackServiceProvider.php

```diff
@@ -4,6 +4,7 @@
 
 use Backpack\CRUD\app\Http\Middleware\ThrottlePasswordRecovery;
 use Backpack\CRUD\app\Library\CrudPanel\CrudPanel;
+use Backpack\CRUD\app\Library\CrudPanel\CrudPanelViewNamespaces;
```

Comment:
Wouldn't this be a little shorter?

```suggestion
use Backpack\CRUD\app\Library\ViewNamespaces;
```

Also, now I regret not calling them `locations` instead of `namespaces` 🤦‍♂️ Would have been much clearer.

Reply:
I agree with both of your statements. 

Will refactor accordingly. 👍 (keeping namespaces)